distuv/norm_example_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv_test
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/gonum/stat"
+	"github.com/gonum/stat/distuv"
+)
+
+func Example_normal() {
+	const (
+		nmax = 1000
+		seed = 1234
+	)
+
+	// Create a normal distribution
```

Comment:
Please use a `nil` source and just don't have an expected output. We don't need this tested.